client/components/login/login.js

```diff
@@ -477,6 +479,7 @@ Login.propTypes = {
   authenticate: PropTypes.func.isRequired,
   verifyMobileNumber: PropTypes.func.isRequired,
   setIsActive: PropTypes.func.isRequired,
+  setUserData: PropTypes.func.isRequired,
```

Comment:
I think that now that we are introducing this, we should be able to get rid of `verifyMobileNumber` and `setIsActive`.
To do that we should make sure that `setUserData` can edit one specific attribute.